lib/bolero-libfuzzer/src/lib.rs

```diff
@@ -140,11 +135,8 @@ pub mod fuzzer {
     #[no_mangle]
     pub unsafe extern "C" fn LLVMFuzzerTestOneInput(data: *const u8, size: usize) -> i32 {
         let data_slice = std::slice::from_raw_parts(data, size);
-        if (TESTFN.as_mut().expect("uninitialized test function"))(data_slice) {
-            0
-        } else {
-            1
-        }
+        (TESTFN.as_mut().expect("uninitialized test function"))(data_slice);
+        0
```

Comment:
This and the below are due to the way test failures get recognized by clusterfuzz. Sorry I forgot to point towards the commit message, where I had explained it 😅 
Here it is:
> LLVMFuzzerTestOneInput is not supposed to ever return something other
than 0. Instead, the whole process should abort in case of a bug being
detected.
> 
> Hence, this replaces the previous returning of 1 (which triggered a
libfuzzer assertion downstream, that was detected by clusterfuzz as the
root cause of the error) by an abort (which lets clusterfuzz detect the
actual cause of the error in the panic message)

Reply:
See also https://www.llvm.org/docs/LibFuzzer.html#fuzz-target ; where values other than 0 and -1 are "reserved for future use"